src/Swashbuckle.AspNetCore.Filters/Examples/ExamplesConverter.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+using Swashbuckle.AspNetCore.Filters.Extensions;
+
+namespace Swashbuckle.AspNetCore.Filters
+{
+    internal class ExamplesConverter
```

Comment:
This code was mostly extracted from `RequestExample` and `ResponseExample` classes. Now it's much less duplicated, however definitely more can be done in this topic.